spec/factories.rb

```diff
@@ -1,20 +1,20 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email) { |n| "person#{n}@example.com" }
-    first_name 'Jone'
-    last_name 'Smith'
+    sequence(:email) { |n| "user#{n}@apiaddict.xyz" }
+    first_name 'Richie'
+    last_name 'Cunningham'
     password 'password'
     password_confirmation 'password'
   end
   factory :api do
-    sequence(:name) { |n| "person#{n} api" }
-    url "www.wunderground.com/something123456789"
-    description "This is a weather API that shows people weather."
+    sequence(:name) { |n| "API ##{n}" }
+    sequence(:url)  { |n| "www.apiaddict.xyz/api#{n}" }
+    description 'This is an awesome paid API.'
     paid false
     user_id 1
```

Comment:
I know this isn't part of your PR, but, you should create a user association here instead of specifing an id.